File: app/src/main/java/com/nowfloats/webactions/models/Extra.java
package com.nowfloats.webactions.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Extra {

    @SerializedName("CurrentIndex")
    @Expose
    private Integer currentIndex;
    @SerializedName("TotalCount")
    @Expose
    private Integer totalCount;
    @SerializedName("PageSize")
    @Expose
    private Integer pageSize;

    public Integer getCurrentIndex() {
        return currentIndex;
    }

    public void setCurrentIndex(Integer currentIndex) {
        this.currentIndex = currentIndex;
    }

    public Integer getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(Integer totalCount) {
        this.totalCount = totalCount;
    }

    public Integer getPageSize() {
        return pageSize;
    }

    public void setPageSize(Integer pageSize) {
        this.pageSize = pageSize;
    }

}

File: appService/src/main/java/com/appservice/model/kycData/Extra.kt
package com.appservice.model.kycData


import com.google.gson.annotations.SerializedName

data class Extra(
    @SerializedName("CurrentIndex")
    var currentIndex: Int? = null,
    @SerializedName("PageSize")
    var pageSize: Int? = null,
    @SerializedName("TotalCount")
    var totalCount: Int? = null
)